### PR TITLE
Modify README

### PR DESCRIPTION
**Pull request by uni-kakurenbo**
Improved contents of the `README`.
- Adjusted the usage example.
- Acknowledgements noted.
- Added a brief summary of the library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +33,19 @@ client.on("message", message => {
 
 client.login('<username>', '<password>');
 ```
+## License
+This library is licensed under the **Apache License 2.0**.
+
+## Acknowledgements
```
Comment (itta611) [inline README.md:39]: I think `credits` is good.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,16 @@
-# Scratch-API-Librarys
+# ScratchAPI.js
+The Scratch API Library
```
Comment (itta611) [inline README.md:2]: Please fix to `A great library for using Scratch API`.

**Review by itta611 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (uni-kakurenbo) [reply README.md:39]: I feel that the word "credit" contains a **legal** "copyright" nuance.
That acknowledgement does not include that meaning.

File: README.md (hunk 2 above)
Comment (uni-kakurenbo) [reply README.md:2]: I think `A great library for **the** Scratch API` is better.

Comment (itta611) [reply README.md:2]: yeah, that's more better!

File: README.md (hunk 1 above)
Comment (itta611) [reply README.md:39]: okay.

**Comment by uni-kakurenbo** [comment]
Is it possible to merge?

**Comment by itta611** [comment]
ok

**Review by itta611 — APPROVED** [review]

**Pull request merged**